### ISSUE TITLE
SharePoint new list experience Issues/Bugs

### ISSUE THREAD

**Issue by juliemturner** [SharePoint/sp-dev-docs#9910]
Per the suggestion of @reedpamsft I'm submitting a series of issues that we've found while working with the new list experience in SharePoint. There are more but they're somewhat intermittent and harder to pin down. It would be ideal if there were a way for users to submit issues with the UX/UI more easily, but I do very much appreciate your openness to having these posted here. 

1. When filtering on a column, the filterable values to select from are limited to values in the list currently loaded into memory. See video attached below
2. View names in the horizontal tabs do not show the entire name of the view in some cases and the title area cannot be resized. This can cause issues when the view name is lengthy where the differentiator of the view name is at the end of the title
3. No way to choose what order the views how up in and they are not alphabetical.
4. Similar to issue 1, If there are many (over 500) items in a list and you are trying to delete them. If you scroll down and select all and delete, the items are deleted. The view says there are no item in the list but there are still hundreds remaining and you need to refresh the page to see them.

Video detailing issue 1
https://github.com/user-attachments/assets/6615ec39-f719-4102-9492-b4c5440edc87

Please feel free to reach out to myself or @sympmarc for more details.

**Comment by reedpamsft** [comment]
@juliemturner thanks for the details!  There should be a button in the UI (top right corner; "Provide feedback to Microsoft") that works to submit issues; let me know here if something about it isn't working.

Anyway, I'll work on getting these assigned to the right folks on the team to look into them.

**Comment by aaclage** [comment]
Any update on this?

Dear @tpodugu-ms, @arkogupta, @jgupta-msft, this issue is also under current support or in backlog tasks for Lists?

**Comment by aaclage** [comment]
@jgupta-msft Thank you for the update and support on related topics.

Best regards,

**Comment by Ashlesha-MSFT** [comment]
Hello @juliemturner,
Thank you for bringing this issue to our attention. We will look into it and get back to you shortly.

**Comment by Ashlesha-MSFT** [comment]
@juliemturner,
As the bug has already been raised and acknowledged, 
could you please confirm if it has been resolved

**Comment by sympmarc** [comment]
@Ashlesha-MSFT - If the bugs were indeed recorded as such internally at Microsoft and someone worked on them, we never heard anything about it. Wouldn't it make more sense to see where they are on the internal queue before asking us to test them again? There isn't one bug above, but multiple bugs.

**Comment by Ashlesha-MSFT** [comment]
@sympmarc,

Sorry for delayed responce!

I’ve tested the reported issues in my tenant using a list with 400+ rows. Based on my testing, the other issues could not be reproduced, but Issue 2 still persists — view names in the horizontal tabs are getting truncated when they’re long. While we can hover to see the full name, the lack of resizing or wrapping still affects usability, especially when the differentiating part of the view name is at the end.

We've also reached out to the engineering team regarding this issue. In the meantime, could you please check on your side and confirm if the behavior still exists in your environment as well?

**Comment by VesaJuvonen** [comment]
Let's look in @reedpamsft as the owning engineering lead for an update if the issues are still there. Good to get things addressed for sure, so that we can close fixed issues as needed or - create new issues for tracking.

**Comment by sympmarc** [comment]
@Ashlesha-MSFT - With the delay in the time we reported these things and now, it's hard to know exactly what you want us to test. If you could be more specific, I'd be willing to test for you. At the time, @juliemturner and I had specific test scenarios easily at hand. Now we'll need to create them or find places where they can be demonstrated.

It's truly sad to me that we're 4 months down the road from legitimate bug reports and we're just getting a response. If some of these bugs were indeed fixed, wouldn't you have a log of that? Shouldn't you have asked us to test when the fixes were each rolled out?

**Wait - it's been 11 months!!!!!**

**Comment by reedpamsft** [comment]
@VesaJuvonen right; per my earlier comments these were discussed internally with engineering owners back when they were first brought to our attention.  Number 1 pre-dates the list redesign and has to do with the way filters are stored and fetched; 2 and 3 are not possible given the way that list metadata storage works on the back-end, and 4 is a tricky race condition; we have been working on the way that list pages are fetched and have fixed what we could repro internally.

**Comment by juliemturner** [comment]
@reedpamsft #2 (horizontal tabs cut of the view name) above was just fine until the powers that be decided that SharePoint lists should be replaced with Microsoft Lists. So "not possible" is not the right answer. It's obviously completely possible to fix the UI so that it doesn't do this.

**Comment by reedpamsft** [comment]
@juliemturner  right.  The view tabs are part of the visual redesign, which was intentional, and limiting the length of the view tab names is also an intentional part of that change.

Allowing more detailed configuration of the view tabs for people who want that is the part that is technically difficult, but there should be a change rolling out soon which allows list owners to limit the number of view tabs shown to 1 (the current active tab) which might somewhat address concerns 2 and 3.
